### PR TITLE
Update Arabic Form Translations

### PR DESCRIPTION
**Pull request by zairigimad**
| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | no
| New feature?  | no 
| Deprecations? | no 
| Tickets       | Fix #... 
| License       | MIT
| Doc PR        | symfony/symfony-docs#... 

This is related to https://github.com/symfony/symfony/issues/38723

### CODE REVIEW

File: src/Symfony/Component/Form/Resources/translations/validators.ar.xlf [hunk 1]
```diff
@@ -14,6 +14,126 @@
...
+                <source>Please select a valid country.</source>
+                <target>الرجاء اختيار بلد صالح.</target>
+            </trans-unit>
+            <trans-unit id="105">
+                <source>Please select a valid currency.</source>
+                <target>الرجاء اختيار عملة صالحة.</target>
+            </trans-unit>
+            <trans-unit id="106">
+                <source>Please choose a valid date interval.</source>
+                <target>الرجاء اختيار فاصل زمني صالح.</target>
```
Comment (Nyholm) [inline src/Symfony/Component/Form/Resources/translations/validators.ar.xlf:47]: Does this translation really say "a valid **date** interval"? Or is it just "interval"?

File: src/Symfony/Component/Form/Resources/translations/validators.ar.xlf [hunk 2]
```diff
@@ -14,6 +14,126 @@
...
+                <source>Please enter an integer.</source>
+                <target>الرجاء إدخال عدد صحيح.</target>
+            </trans-unit>
+            <trans-unit id="112">
+                <source>Please select a valid language.</source>
+                <target>الرجاء اختيار لغة صالحة.</target>
+            </trans-unit>
+            <trans-unit id="113">
+                <source>Please select a valid locale.</source>
+                <target>الرجاء اختيار موقع صالح.</target>
```
Comment (Nyholm) [inline src/Symfony/Component/Form/Resources/translations/validators.ar.xlf]: This should say "locale" as in language (en, fr, ar), not "location"

File: src/Symfony/Component/Form/Resources/translations/validators.ar.xlf [hunk 3]
```diff
@@ -14,6 +14,126 @@
...
+                <source>Please enter a valid time.</source>
+                <target>الرجاء إدخال وقت صالح.</target>
+            </trans-unit>
+            <trans-unit id="120">
+                <source>Please select a valid timezone.</source>
+                <target>الرجاء تحديد منطقة زمنية صالحة.</target>
+            </trans-unit>
+            <trans-unit id="121">
+                <source>Please enter a valid URL.</source>
+                <target>أدخل رابط صحيح من فضلك.</target>
```
Comment (Nyholm) [inline src/Symfony/Component/Form/Resources/translations/validators.ar.xlf]: Make sure this translate to "URL" and not "link". It is technically a difference.

File: src/Symfony/Component/Form/Resources/translations/validators.ar.xlf [hunk 4]
```diff
@@ -14,6 +14,126 @@
...
+                <source>Please enter a valid search term.</source>
+                <target>الرجاء إدخال مصطلح البحث ساري المفعول.</target>
+            </trans-unit>
+            <trans-unit id="123">
+                <source>Please provide a valid phone number.</source>
+                <target>يرجى تقديم رقم هاتف صالح.</target>
+            </trans-unit>
+            <trans-unit id="124">
+                <source>The checkbox has an invalid value.</source>
+                <target>خانة الاختيار لها قيمة غير صالحة.</target>
```
Comment (Nyholm) [inline src/Symfony/Component/Form/Resources/translations/validators.ar.xlf:119]: Could you confirm that this is correct?

**Review by Nyholm — CHANGES_REQUESTED** [review]
Lovely. You are the best!

I have a few comments and questions.

File: src/Symfony/Component/Form/Resources/translations/validators.ar.xlf (hunk 1 above)
Comment (zairigimad) [reply src/Symfony/Component/Form/Resources/translations/validators.ar.xlf:47]: yes in Arabic the "date interval" is translated to "فاصل زمني" where "interval" is  "فاصل" and "زمني" refers to "date" :)

File: src/Symfony/Component/Form/Resources/translations/validators.ar.xlf (hunk 4 above)
Comment (zairigimad) [reply src/Symfony/Component/Form/Resources/translations/validators.ar.xlf:119]: yes that is correct :)

File: src/Symfony/Component/Form/Resources/translations/validators.ar.xlf (hunk 2 above)
Comment (zairigimad) [reply src/Symfony/Component/Form/Resources/translations/validators.ar.xlf]: updated to be more clear.

File: src/Symfony/Component/Form/Resources/translations/validators.ar.xlf (hunk 3 above)
Comment (zairigimad) [reply src/Symfony/Component/Form/Resources/translations/validators.ar.xlf]: I've just updated this to be more precise that is a URL

File: src/Symfony/Component/Form/Resources/translations/validators.ar.xlf (hunk 4 above)
Comment (Nyholm) [reply src/Symfony/Component/Form/Resources/translations/validators.ar.xlf:119]: Thank you

File: src/Symfony/Component/Form/Resources/translations/validators.ar.xlf (hunk 3 above)
Comment (Nyholm) [reply src/Symfony/Component/Form/Resources/translations/validators.ar.xlf]: Google translates this to `Enter a valid address, please.`, I assume Google is wrong and it really says "URL", right?

File: src/Symfony/Component/Form/Resources/translations/validators.ar.xlf (hunk 1 above)
Comment (Nyholm) [reply src/Symfony/Component/Form/Resources/translations/validators.ar.xlf:47]: Thank you

File: src/Symfony/Component/Form/Resources/translations/validators.ar.xlf [hunk 5]
```diff
@@ -14,6 +14,126 @@
...
+                <source>Please enter an integer.</source>
+                <target>الرجاء إدخال عدد صحيح.</target>
+            </trans-unit>
+            <trans-unit id="112">
+                <source>Please select a valid language.</source>
+                <target>الرجاء اختيار لغة صالحة.</target>
+            </trans-unit>
+            <trans-unit id="113">
+                <source>Please select a valid locale.</source>
+                <target>الرجاء اختيار لغة صالحة.</target>
```
Comment (Nyholm) [inline src/Symfony/Component/Form/Resources/translations/validators.ar.xlf:75]: But now it is the same as 112. It says "language". 

A language is "Swedish", "English", "German". But a locale is "sv", "en", "de". 

Are you sure this is correct?

File: src/Symfony/Component/Form/Resources/translations/validators.ar.xlf (hunk 3 above)
Comment (zairigimad) [reply src/Symfony/Component/Form/Resources/translations/validators.ar.xlf]: yes it says URL, I can be more precise by combining the previous and the newest update  and the previous one "عنوان الرابط" , google Translate now happy `Please enter a valid URL`

Comment (Nyholm) [reply src/Symfony/Component/Form/Resources/translations/validators.ar.xlf]: Google translate is my only way to check if this is correct. 

Im not saying that Google translate is correct, but it helps me to see what items I should ask you to confirm the correctness. 

What ever translation that says "URL" and not "link" and that sounds good, it a good translation. If you are happy with the current one that so am I.

Comment (zairigimad) [reply src/Symfony/Component/Form/Resources/translations/validators.ar.xlf]: Yes, i understand, but yes like you said Google translate is not always correct.
This version of translation is more precise and clear, it’s good for me :)

File: src/Symfony/Component/Form/Resources/translations/validators.ar.xlf (hunk 5 above)
Comment (zairigimad) [reply src/Symfony/Component/Form/Resources/translations/validators.ar.xlf:75]: Yes in Arabic there is no distinction between the two. The version before was about location. But location is not what we want here

File: src/Symfony/Component/Form/Resources/translations/validators.ar.xlf (hunk 3 above)
Comment (Nyholm) [reply src/Symfony/Component/Form/Resources/translations/validators.ar.xlf]: Excellent!

Thank you.

File: src/Symfony/Component/Form/Resources/translations/validators.ar.xlf (hunk 5 above)
Comment (Nyholm) [reply src/Symfony/Component/Form/Resources/translations/validators.ar.xlf:75]: Cool cool cool

**Review by Nyholm — APPROVED** [review]
Thank you. 

I am happy with this :+1:

**Review by derrabus — APPROVED** [review]

**Comment by derrabus** [comment]
Thank you @zairigimad.

**Pull request merged**